.github/workflows/agw.branch_push.yml

```
@@ -0,0 +1,52 @@
+name: push-to-branch
+on:
+  push:
+    branches:
+      - "**"
+      - "!master"
```

Shouldn't it be `main` instead of `master`?
```suggestion
      - "!main"
```